item-impl/src/main/java/com/example/auction/item/impl/PItemCommand.java

```diff
@@ -43,6 +41,92 @@ public int hashCode() {
         }
     }
 
+
+    final class UpdateItem implements PItemCommand, PersistentEntity.ReplyType<PUpdateItemResult> {
+        private final UUID id;
+        private final UUID creator;
+        final String title;
+        private final String description;
+        private final String currencyId;
+        private final int increment;
+        private final int reservePrice;
+        private final Duration auctionDuration;
+
+        @JsonCreator
+        public UpdateItem(UUID id, UUID creator, String title, String description, String currencyId, int increment, int reservePrice, Duration auctionDuration) {
```

Comment:
We are really duplicating the `Item`/`PItem` details into a lot of different places, which is going to make it very painful to add fields. I wonder whether we should have the command simply wrap a PItem derived from the item provided in the request. That's how `CreateItem` works.